content/en/blogs/faas-openfunction.md

```diff
@@ -0,0 +1,300 @@
+---
+title: 'OpenFunction: Build a Novel Platform for Open-Source Function Computing from Scratch'  
+tag: 'Serverless, FaaS, OpenFunction'  
+keywords: Serverless, FaaS, OpenFunction  
+description: This article describes function computing concepts, reference architectures, and the latest trends in the cloud-native Serverless field, and further explains how OpenFunction builds a novel platform for open-source function computing from scratch.   
+createTime: '2021-12-03'  
+author: 'Huo Bingjie, Felix'  
+snapshot: 'https://pek3b.qingstor.com/kubesphere-community/images/serverless-faas-openfunction.png'
+---
+
+**Serverless computing**, commonly known as Serverless, has become a buzzword in the cloud-native field. It will fuel the next wave of development in cloud computing after IaaS and PaaS. Serverless emphasizes an architecture philosophy and service model that allows developers to focus on implementing business logics in applications rather than on managing infrastructures (e.g., servers). In its paper *Cloud Programming Simplified: A Berkeley View on Serverless Computing*, the University of California at Berkeley presents two key viewpoints on Serverless:
+
++ Although serverful computing will be here to stay, its importance will  decrease as Serverless computing continues to mature.
++ Serverless computing will eventually become the paradigm of the cloud computing age, replacing the serverful computing model to a  great extent and putting an end to the Client-Server age.
```

Comment:
Please check if there is any specific term for Serverful Computing as Google SERP shows no matched content. Thanks.

Reply:
This term appeared in [the Berkeley paper](https://www2.eecs.berkeley.edu/Pubs/TechRpts/2019/EECS-2019-3.pdf). If there is no matched content in SERP, how about change it to `server-centric` or `server-based`?

![image](https://user-images.githubusercontent.com/75110798/147800139-d646957a-dae2-49fe-a105-cb0bcfef3863.png)

Reply:
server-based seems more common

Reply:
Use the original word from the paper is ok